examples/script/google2016_unbreakable/win.py

```diff
@@ -51,8 +51,7 @@ def hook(state):
 def hook(state):
     print("Hit the final state.. solving")
 
-    buffer = state.cpu.read_bytes(input_addr, num_bytes)
-    res = ''.join(chr(state.solve_one(x)) for x in buffer)
+    res = ''.join(map(chr, state.solve_buffer(input_addr, num_bytes)))
```

Comment:
now that we have manticore-examples, we can remove this script and directory entirely? we can update the script there, and maybe edit the docstring to reflect that the original submitted script has been modified